tidb-cloud/data-service-manage-endpoint.md

```diff
@@ -118,7 +118,11 @@ On the right pane of the endpoint details page, you can click the **Properties**
 
 - **Cache Response**: this property is available only when the request method is `GET`. When **Cache Response** is enabled, TiDB Cloud Data Service can cache the response returned by your `GET` requests within a specified time-to-live (TTL) period.
 - **Time-to-live (s)**: this property is available only when **Cache Response** is enabled. You can use it to specify the time-to-live (TTL) period in seconds for cached response. During the TTL period, if you make the same `GET` requests again, Data Service returns the cached response directly instead of fetching data from the target database again, which improves your query performance.
```

Comment:
```suggestion
- **Time-to-live(s)**: this property is available only when **Cache Response** is enabled. You can use it to specify the time-to-live (TTL) period in seconds for cached response. During the TTL period, if you make the same `GET` requests again, Data Service returns the cached response directly instead of fetching data from the target database again, which improves your query performance.
```
<img width="310" alt="image" src="https://github.com/pingcap/docs/assets/60599231/0dd96dd1-51e7-4c5c-8bef-66df725efdc1">